minigotchi-ESP32/deauth.cpp

```diff
@@ -217,7 +217,11 @@ bool Deauth::select() {
       Display::updateDisplay(
           "('-')",
           "Selected AP is in the whitelist. Skipping deauthentication...");
-      Parasite::sendDeauthStatus(SKIPPING_WHITELIST);
+      < < < < < < < HEAD delay(1000);
```

Comment:
We should probably use the int's `Config::longDelay`(1000) and `Config::shortDelay`(500) for delays just to make the delays consistent.

Reply:
Delays fixed.